hibernate-core/src/test/java/org/hibernate/test/legacy/MultiTableTest.java

```diff
@@ -392,6 +396,7 @@ public void testMultiTable() throws Exception {
 	}
 
 	@Test
+	@RequiresDialectFeature(DialectChecks.SupportNoWait.class)
```

Comment:
can this be changed with `@SkipForDialect( value = CockroachDB192Dialect.class`

Reply:
done